test/integration/helpers/project.js

```diff
@@ -61,6 +61,8 @@ exports.deleteAllProjects = function() {
     browser.wait(protractor.ExpectedConditions.elementToBeClickable(deleteButton), 2000);
     deleteButton.click();
     h.waitForPresence(".alert-success", "marked for deletion");
+    // dont click next until we know the modal disappeared
+    browser.sleep(1000);
```

Comment:
Is it better to have a `waitForRemoval` or similar and wait for the DOM element to disappear?